reddit_liveupdate/validators.py

```diff
@@ -28,6 +30,19 @@ def run(self, id):
             return None
 
 
+class VLiveUpdateEventUrl(VLiveUpdateEvent):
+    def run(self, url):
+        if not url:
+            return None
+
+        u = UrlParser(url)
+        id = re.match(r'/live/(\w+)/?', u.path)
```

Comment:
Yeah, good idea ( 82c26ac ).  I was going to add some unit tests here, too, but there are some hiccups to getting that working and I figured it wasn't a big deal.